.github/workflows/building-bundle.yml

```diff
@@ -34,3 +39,33 @@ jobs:
       with:
         name: vshexa-${{ github.sha }}
         path: '*.vsix'
+
+  deploy-all:
+    name: 'Deploying'
+    needs: build-all
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ windows-latest ]
+        node-version: [ 16.x ]
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0
+
+    - name: Setup Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+
+    - name: Publish Hexa VSCode bundle
+      if: startsWith(github.ref, 'refs/tags/releases/')
```

Comment:
Confirming that this works as intended

Reply:
@PeyTy this option is recommended in the extension deployment documentation